deliverable_setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+read -p "Which CI/CD service do you use (Can be edited later) [(g)ithub/(b)itrise/(l)ater]: " ciService
+
+if [ "$ciService" = "g" -o "$ciService" = "github" ]; then
```

Comment:
I am a bit confused about the syntax inside the `if []` . Are we checking if "$ciService" equal to "g" ? If we check equality then should we use `==` instead of `=`?

Reply:
[ "a" = "b" ] and [ "a" == "b" ] is the syntax for checking string equal.
> also... "a"="b" is the syntax for assigning value. 🙈 

https://tldp.org/LDP/abs/html/comparison-ops.html

Reply:
@blyscuit thanks , I thought one `=`  means assignment

Reply:
@Shayokh144 It's the space that's matter 😆